### PR TITLE
Feature 14 user story system monitoring tool documentation

### PR DESCRIPTION
**Pull request by Bansikah**
Documentation on monitoring tools user story 14

### CODE REVIEW

File: services/__pycache__/time_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/time_service.cpython-310.pyc:1]: Please remove this file, as it doesn't have a thing to do with your current ticket

File: services/__pycache__/memory_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/memory_service.cpython-310.pyc:1]: Please remove this file

File: services/__pycache__/cpu_service.cpython-310.pyc
Comment (stephane-segning) [inline services/__pycache__/cpu_service.cpython-310.pyc:1]: Please remove this file

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Perfect! Just these 3 points...

File: services/__pycache__/memory_service.cpython-310.pyc
Comment (Bansikah) [reply services/__pycache__/memory_service.cpython-310.pyc:1]: ok sir, right away

**Review by stephane-segning — APPROVED** [review]
Good!

**Pull request merged**